etc/profile-a-l/coyim.profile

```diff
@@ -0,0 +1,46 @@
+# Firejail profile for coyim
+# Description: GTK Jabber client written in Go
+# This file is overwritten after every install/update
+# Persistent local customizations
+include coyim.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/coyim
```

Comment:
`blacklist ${HOME}/.config/coyim` needs to be added to `etc/disable-programs.inc`